style.css

```diff
@@ -0,0 +1,586 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap');
+*{
+    box-sizing: border-box;
+    padding: 0px;
+    margin: 0px;
+    font-family: 'Poppins', sans-serif;
+}
+
+.container{
+    min-height: 100vh;
+    padding: 10px 133px;
+}
+
+/* Header section starts */
+.header{
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.header .logo{
+    width: 75px;
+    height: 102px;
+}
+.nav-links ul{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    list-style: none;
+    gap:2.5rem;
+}
+.nav-links ul li a{
+    font-weight: 500;
+    font-size: 0.93rem;
+    color: black;
+    text-decoration: none;
+}
+.nav-links button{
+    padding: .3rem 1rem;
+    font-weight:500;
+    background-color: #fff;
+    border-radius: 5px;
+    border:1.2px solid black;
+}
+.nav-links select{
+   border: none;
+   outline: none;
+   font-weight:500;
+}
+/* header section Closed */
+
+/* banner section starts */
+.banner{
+    display: flex;
+    width: 100%;
+    min-height: 70vh;
+}
+.banner__text{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 50%;
+}
+.banner__text--quote{
+    color: rgb(227, 93, 45);
+    font-weight: 600;
+}
+.banner__text--big-text{
+    padding: .8rem 0rem;
+    line-height: 4.3rem;
+    font-size: 3.75rem;
+    word-spacing: 0.62rem;
+    color: #181E4B;
+    font-weight: 900;
+}
+.banner__text--about{   
+    padding-right: 3.3rem;
+    line-height: 1.56rem;
+    color: rgb(112, 111, 111);
+    font-size: 0.93rem;
+}
+.banner__text--btn{
+    width: 9.375rem;
+    margin-top: 0.93rem;
+    font-size: 1rem;
+    height: 3.75rem;
+    border-radius: 10px;
+    font-weight: 500;
+    border: none;
+    cursor: pointer;
+    background: rgb(245, 171, 33);
+    color: #fff;
+}
+.banner__image{
+    width: 55%;
+    height: 100%;
+}
+.banner__image img{
+    height: 100%;
+    width: 120%;
+    margin-left:-6.25rem;
+    background-size: contain;
+}

```

Comment:
Consider using background-image for the .banner__image img instead of setting its width to 120% and adjusting the margin to position it. This can lead to unexpected behavior on different devices.